src/lib/components/Nav.svelte

```diff
@@ -9,18 +9,17 @@
   @use "../styles/mixins/media";
 
   div {
-    padding: var(--padding-2x) var(--padding-2x) 0;
-    margin: 0;
+    // title is hidden on small devices
```

Comment:
Did you decide to make this change for all clients?

Reply:
Yes, pretty bold move to avoid extra complexity. The solution with `{#if $slot.title...` wouldn't work, because it depends on the breakpoint and it's not allowed to wrap `<div slot="...` into conditional operators.

Reply:
The title is back.